tests/app/utilities/test_factory.py

```diff
@@ -32,5 +32,10 @@ def test_register_all(self):
         self.assertIsInstance(factory.create("test-a"), TestClassA)
         self.assertIsInstance(factory.create("test-b"), TestClassB)
 
+    def test_create_value_error(self):
```

Comment:
I think you're right, I can't seem to find it being used, 'SubmitterFactory' is used but that's a different thing.